src/Features/LanguageServer/Protocol/Handler/Completion/CompletionHandler.cs

```diff
@@ -54,7 +54,8 @@ public override async Task<LSP.CompletionItem[]> HandleRequestAsync(LSP.Completi
             var completionOptions = documentOptions
                 .WithChangedOption(CompletionOptions.SnippetsBehavior, SnippetsRule.NeverInclude)
                 .WithChangedOption(CompletionOptions.ShowItemsFromUnimportedNamespaces, false)
-                .WithChangedOption(CompletionServiceOptions.IsExpandedCompletion, false);
+                .WithChangedOption(CompletionServiceOptions.IsExpandedCompletion, false)
+                .WithChangedOption(CompletionServiceOptions.DisallowAddingImports, true);
```

Comment:
Why is this option needed here?

Reply:
LSP does not support adding imports.

Reply:
It's mentioned in one of the work items linked above.

Reply:
`ShowItemsFromUnimportedNamespaces = false` above is sufficient, but I guess this doesn't hurt.